<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventDialog</class>
 <widget class="QDialog" name="AddEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Event</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="safety_event">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Event</string>
   </property>
   <item>
    <property name="text">
     <string>Fire Alarm (from elevator)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fire Alarm (from building)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Door Obstruction</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Overload</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Power Outage</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Safety Event</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="elevator_id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="elevator_id_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevator ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timestep">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Timestep of Event:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
